boards/sensebox/include/board.h

```diff
@@ -0,0 +1,156 @@
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name   LED pin definitions and handlers
+ * @{
+ */
+#define LED0_PIN            GPIO_PIN(PA, 27)
+
+#define LED_PORT            PORT->Group[PA]
+#define LED0_MASK           (1 << 27)
+
+#define LED0_ON             (LED_PORT.OUTSET.reg = LED0_MASK)
+#define LED0_OFF            (LED_PORT.OUTCLR.reg = LED0_MASK)
+#define LED0_TOGGLE         (LED_PORT.OUTTGL.reg = LED0_MASK)
+
+#define LED1_PIN            GPIO_PIN(PA, 28)
+#define LED1_MASK           (1 << 28)
+
+#define LED1_ON             (LED_PORT.OUTSET.reg = LED1_MASK)
+#define LED1_OFF            (LED_PORT.OUTCLR.reg = LED1_MASK)
+#define LED1_TOGGLE         (LED_PORT.OUTTGL.reg = LED1_MASK)
+/** @} */
+
+/**
+ * @name    SW0 (Button) pin definitions
+ * @{
+ */
+#define BTN0_PORT           PORT->Group[PA]
+#define BTN0_PIN            GPIO_PIN(PA, 20)
+#define BTN0_MODE           GPIO_IN_PU
+/** @} */
+
+/**
+ * @name    XBEE1 bus
+ * @brief   XBEE1 is enabled by default. The enable pin is
+ * 'XBEE1_EN_PIN'. To disable this bus call 'XBEE1_DISABLE', to re-enable it
+ * call 'XBEE1_ENABLE'.
+ * @{
+ */
+#define XBEE1_EN_PORT           PORT->Group[PB]
+#define XBEE1_EN_MASK           (1 << 3)
+#define XBEE1_EN_MODE           GPIO_OUT
+#define XBEE1_EN_PIN            GPIO_PIN(PB, 3)
+
+#define XBEE1_ENABLE             (XBEE1_EN_PORT.OUTCLR.reg = XBEE1_EN_MASK)
+#define XBEE1_DISABLE            (XBEE1_EN_PORT.OUTSET.reg = XBEE1_EN_MASK)
+
+#define XBEE1_CS_PIN            GPIO_PIN(PA, 18)
+
+#define XBEE1_INT_PIN           GPIO_PIN(PA, 21)
+/** @} */
+
+/**
+ * @name    XBEE2 bus
+ * @brief   XBEE2 is enabled by default. The enable pin is
+ * 'XBEE2_EN_PIN'. To disable this bus call 'XBEE2_DISABLE', to re-enable it
+ * call 'XBEE2_ENABLE'.
+ * @{
+ */
+#define XBEE2_EN_PORT           PORT->Group[PB]
+#define XBEE2_EN_MASK           (1 << 10)
+#define XBEE2_EN_PIN            GPIO_PIN(PB, 10)
+#define XBEE2_EN_MODE           GPIO_OUT
+
+#define XBEE2_ENABLE             (XBEE2_EN_PORT.OUTCLR.reg = XBEE2_EN_MASK)
+#define XBEE2_DISABLE            (XBEE2_EN_PORT.OUTSET.reg = XBEE2_EN_MASK)
+
+#define XBEE2_CS_PIN            GPIO_PIN(PA, 14)
+
+#define XBEE2_INT_PIN           GPIO_PIN(PA, 15)
+/** @} */
+
+/**
+ * @name    I2C bus
+ * @brief   I2C is enabled by default. The enable pin is
+ * 'I2C_EN_PIN'. To disable this bus call 'I2C_DISABLE', to re-enable it call
+ * 'I2C_ENABLE'.
+ * @{
+ */
+#define I2C_EN_PORT           PORT->Group[PB]
+#define I2C_EN_MASK           (1 << 11)
+#define I2C_EN_PIN            GPIO_PIN(PB, 11)
+#define I2C_EN_MODE           GPIO_OUT
+
+#define I2C_ENABLE             (I2C_EN_PORT.OUTSET.reg = I2C_EN_MASK)
+#define I2C_DISABLE            (I2C_EN_PORT.OUTCLR.reg = I2C_EN_MASK)
+/** @} */
+
+
+
+/**
+ * @name SX127X
```

Comment:
Do you have a link to the sx127x module you want to use on the Xbee port ? This sounds weird to me.